apps/portal/js/create.js

```diff
@@ -148,12 +148,15 @@ $(function () {
     // Bind event handlers for dashboard ID field
     $('#ues-dashboard-id').on("keypress", function (e) {
         return sanitizeOnKeyPress(this, e, /[^a-z0-9-\s]/gim);
-    }).on('keyup', function () {
+    }).on('keyup', function (e) {
```

Comment:
better be consistent with quotations for defining strings, use double quotes for outer strings and single quotes for inner strings.